packages/create-app/templates/default-app/app-config.production.yaml

```diff
@@ -28,6 +28,10 @@ backend:
       #   ca: # if you have a CA file and want to verify it you can uncomment this section
       #     $file: <file-path>/ca/server.crt
 
+auth:
+  providers:
+    guest: null
```

Comment:
Switched to this method instead. The `app-config.local.yaml` works well for a single user, but it kinda breaks when you bring more users along, because gitignore'd by default